documentation/chatmeleon-swagger.yaml

```diff
@@ -337,6 +540,47 @@ components:
             $ref: "#/components/schemas/ObjectId"
             description: List of message IDs the user has seen
 
+    Account:
+      type: object
+      properties:
+        id:
+          type: string
+          pattern: "^[0-9a-f]{24}$"
+          description: MongoDB _id of the account.
+        userId:
+          type: string
+          pattern: "^[0-9a-f]{24}$"
+          description: User ID
+        type:
+          type: string
+          description: Type of account (e.g., "google", "facebook")
```

Comment:
Maybe define an enum for this instead of documenting it as arbitrary strings? Ignore this comment if these account types depend on third party definitions I guess.